ray-operator/controllers/ray/common/pod.go

```diff
@@ -1038,3 +1042,48 @@ func isGPUResourceKey(key string) bool {
 	match, _ := regexp.MatchString(`nvidia\.com/mig-\d+g\.\d+gb$`, key)
 	return match
 }
+
+// addDefaultRayNodeLabels passes default Ray node labels to Ray runtime environment
+func addDefaultRayNodeLabels(pod *corev1.Pod) {
+	pod.Spec.Containers[utils.RayContainerIndex].Env = append(
+		pod.Spec.Containers[utils.RayContainerIndex].Env,
+		// used to set the ray.io/market-type node label
+		corev1.EnvVar{
+			Name:  "RAY_NODE_MARKET_TYPE",
```

Comment:
Move `"RAY_NODE_MARKET_TYPE"`,  `"RAY_NODE_ZONE"`, and `"RAY_NODE_REGION"` to `constant.go`.